src/runtime/keyed-crypto.ts

```diff
@@ -195,7 +195,7 @@ class noCrypto implements CryptoAction {
       id: sthis.nextId().str,
       name: "noCrypto",
       get: () => {
-        throw this.logger.Error().Msg("noCrypto.get not implemented").AsError();
+        return Promise.resolve(undefined); // No specific key for noCrypto
```

Comment:
not sure this is right... I think we want to point to the 'insecure' key here? or maybe this code path should not exercise when key == 'insecure' in which case we can revert this line. investigating.